subsys/fs/fuse_fs_access.c

```diff
@@ -66,7 +66,7 @@ static bool is_mount_point(const char *path)
 {
 	char dir_path[PATH_MAX];
 
-	sprintf(dir_path, "%s", path);
+	snprintk(dir_path, sizeof(dir_path), "%s", path);
```

Comment:
note that this function is called from the host machine and not in zephyr context

Reply:
Removed